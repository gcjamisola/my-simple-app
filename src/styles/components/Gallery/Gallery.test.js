import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { GalleryContainer, GalleryImage } from './index';

test('it works', () => {
	let tree = renderer.create(<GalleryContainer/>).toJSON();
	expect(tree).toMatchSnapshot();

	tree = renderer.create(<GalleryImage/>).toJSON();
	expect(tree).toMatchSnapshot();
});
