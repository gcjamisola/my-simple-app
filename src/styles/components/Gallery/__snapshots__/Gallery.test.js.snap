// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

<div
  className="c0"
/>
`;

exports[`it works 2`] = `
.c0 {
  width: 10%;
}

<img
  className="c0"
/>
`;
